docker/Dockerfile.aarch64-linux-android

```diff
@@ -34,15 +34,21 @@ RUN /android-system.sh arm64
 
 ENV CROSS_TOOLCHAIN_PREFIX=aarch64-linux-android-
 ENV CROSS_SYSROOT=/android-ndk/sysroot
+ENV CROSS_ANDROID_SDK=$ANDROID_SDK
 COPY android-symlink.sh /
 RUN /android-symlink.sh aarch64 aarch64-linux-android
 
 COPY android-runner /
 
+COPY android.cmake /opt/android.cmake
+ENV CMAKE_TOOLCHAIN_FILE=/opt/android.cmake
+ENV CROSS_CMAKE_OBJECT_FLAGS="-DANDROID -ffunction-sections -fdata-sections -fPIC"
+
 # Libz is distributed in the android ndk, but for some unknown reason it is not
 # found in the build process of some crates, so we explicit set the DEP_Z_ROOT
+ENV CROSS_TARGET_RUNNER="/android-runner aarch64"
```

Comment:
This allows us to use it both for the cargo runner and for `CMAKE_CROSSCOMPILING_EMULATOR`, which is required when using `try_run` in some CMake build systems.